database/yugabytedb/yugabytedb.go

```diff
@@ -4,41 +4,50 @@ import (
 	"context"
 	"database/sql"
 	"errors"
-	"fmt"
 	"io"
 	"io/ioutil"
 	"net/url"
 	"regexp"
 	"strconv"
+	"time"
 
+	"github.com/cenkalti/backoff/v4"
 	"github.com/golang-migrate/migrate/v4"
 	"github.com/golang-migrate/migrate/v4/database"
 	"github.com/hashicorp/go-multierror"
 	"github.com/lib/pq"
 	"go.uber.org/atomic"
 )
 
-func init() {
-	db := YugabyteDB{}
-	database.Register("yugabyte", &db)
-	database.Register("yugabytedb", &db)
-	database.Register("ysql", &db)
-}
-
-var DefaultMigrationsTable = "migrations"
-var DefaultLockTable = "migrations_locks"
+const (
+	DefaultRetryMaxInterval    = time.Second * 15
+	DefaultRetryMaxElapsedTime = time.Second * 30
+	DefaultRetryMaxRetries     = 10
+	DefaultMigrationsTable     = "migrations"
+	DefaultLockTable           = "migrations_locks"
+)
 
 var (
 	ErrNilConfig          = errors.New("no config")
 	ErrNoDatabaseName     = errors.New("no database name")
 	ErrMaxRetriesExceeded = errors.New("max retries exceeded")
 )
 
+func init() {
+	db := YugabyteDB{}
+	database.Register("yugabyte", &db)
+	database.Register("yugabytedb", &db)
+	database.Register("ysql", &db)
+}
+
 type Config struct {
-	MigrationsTable string
-	LockTable       string
-	ForceLock       bool
-	DatabaseName    string
+	MigrationsTable     string
+	LockTable           string
+	ForceLock           bool
+	DatabaseName        string
+	RetryMaxInterval    time.Duration
```

Comment:
Simplify these names and their corresponding `x-` options and `Default` values. e.g.
`MaxRetryInterval`, `MaxRetryElapsedTime`, `MaxRetries`

Reply:
fixed